processing/src/main/java/io/druid/segment/serde/DictionaryEncodedColumnPartSerde.java

```diff
@@ -143,7 +144,7 @@ public static SerializerBuilder serializerBuilder()
   public static class SerializerBuilder
   {
     private VERSION version = null;
-    private int flags = NO_FLAGS;
+    private int flags = Feature.NO_BITMAP_INDEX.getMask();
```

Comment:
Suggested to make a method `maskOf(Collection<Feature>)`, and `DEFAULT_FEATURES = Collections.singletonList(NO_BITMAP_INDEX)`

Reply:
I made a `STARTING_FLAGS` constant but I didn't do the `maskOf` function- it doesn't seem useful enough to exist at this point.